tools/clang/test/HLSL/ShaderOpArith.xml

```diff
@@ -2101,38 +1775,38 @@
     <Resource Name="U3" Dimension="BUFFER" Width="1024"
               Flags="ALLOW_UNORDERED_ACCESS" InitialResourceState="COPY_DEST"
               Init="Zero" ReadBack="true" />
+    <!-- groupshared output buffers -->
+    <Resource Name="U4" Dimension="BUFFER" Width="256"
+              Flags="ALLOW_UNORDERED_ACCESS" InitialResourceState="COPY_DEST"
+              Init="Zero" ReadBack="true" />
+    <Resource Name="U5" Dimension="BUFFER" Width="1024"
+              Flags="ALLOW_UNORDERED_ACCESS" InitialResourceState="COPY_DEST"
+              Init="Zero" ReadBack="true" />
```

Comment:
These groupshared outputs were moved up since the root atomics tests no longer have any typed buffers so that the verification code can remain common for both